### ISSUE TITLE
[BUG]: .entry('uid')/.asset('uid') doesn't work

### ISSUE THREAD

**Issue by iyerrama29** [contentstack/datasync-filesystem-sdk#2]
Describe the bug
- **.entry(uid)** doesn't work
- **.asset(uid)** doesn't work

**To Reproduce**
Query a content type/asset
In the query, enter the entry's uid/asset's uid

Notice: The result is empty.

**Query**

```ts
Stack.contentType('blog')
  .entry({uid})

// OR

Stack.asset({uid})
```

**Expected behavior**
The query should return an object, if it exists

**Workarounds**
Use raw `.query()` for list-entries OR `.entry()/.asset()` without on singleton-content type

```ts
// example 1: On multiple content type
Stack.contentType('blog')
  .entries()
  .query({uid: {uid}})

// OR

Stack.assets()
  .query({uid: {uid}})
```

```ts
// example 2: On singleton content type
Stack.contentType('blog')
  .entry()

// OR

Stack.asset()
```

Regards,
Ramanathan K

**Comment by iyerrama29** [comment]
**A quick update**

- The above issue's patch will be part of hotfix/1.0.2 release.
- Along with the above, `.schema(uid)` has also been fixed

**Comment by iyerrama29** [comment]
Closing this. The has been fixed in hotfix/1.0.2
